src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/TestFactory.cs

```diff
@@ -0,0 +1,407 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Equinor.ProCoSys.IPO.ForeignApi.LibraryApi.FunctionalRole;
+using Equinor.ProCoSys.IPO.ForeignApi.MainApi.CommPkg;
+using Equinor.ProCoSys.IPO.ForeignApi.MainApi.McPkg;
+using Equinor.ProCoSys.IPO.Infrastructure;
+using Equinor.ProCoSys.IPO.ForeignApi.MainApi.Permission;
+using Equinor.ProCoSys.IPO.ForeignApi.MainApi.Person;
+using Equinor.ProCoSys.IPO.ForeignApi.MainApi.Plant;
+using Equinor.ProCoSys.IPO.WebApi.Middleware;
+using Fusion.Integration.Meeting;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Moq;
+using Newtonsoft.Json;
+
+namespace Equinor.ProCoSys.IPO.WebApi.IntegrationTests
+{
+    public class TestFactory : WebApplicationFactory<Startup>
+    {
+        private readonly string _signerOid = "00000000-0000-0000-0000-000000000001";
+        private readonly string _plannerOid = "00000000-0000-0000-0000-000000000002";
+        private readonly string _preserverOid = "00000000-0000-0000-0000-000000000003";
```

Comment:
should this be called something else than _preserverOid (is it taken from Preservation?)